guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+
+{% include tip/close.html %}
+
+#### Organizing Cashu Metadata Fields
+When designing your Cashu wallet, consider organizing these metadata fields in a logical, user-friendly manner to help users understand and interact with different mints. Here are some suggestions:
+
+- Use the `name` and short `description` in mint listings for quick identification of mints.
+- Display a detailed mint information page using `description_long`, `version`, and `contact` details.
+- Display the `motd` as a prominent notification when relevant to the specific mint.
+- Use the `nuts` information to create a features list that helps users understand each mint's capabilities.
+
+By effectively incorporating these Cashu metadata fields, you can create a more transparent and informative experience for users, helping them make informed decisions about which Cashu mints to trust and use. This is particularly important for Cashu, as users interact directly with individual mints rather than a federation structure like in Fedimint.
+
+### Advance Features
+
+#### Pay to Public Key (P2PK)
+
+{% include image-gallery.html pages = page.images_p2pk %}
+
+[NUT11](https://github.com/cashubtc/nuts/blob/main/11.md) is a powerful feature that allows bitcoin-backed ecash tokens to be securely locked to another user's public key, which is generated by the recipient's wallet. This ensures that only the intended recipient can redeem the ecash. NUT11 enables secure offline payments, preventing double-spending. Beyond these basics, NUT11 supports advanced use cases like timelocks and multisignature (multisig) setups, where ecash can be conditionally spent or jointly owned by multiple parties. When designing make sure these functionalities are clearly communicated to users, highlighting their practical benefits and flexibility.
+
+#### Auto-Swapping to a Default Cashu Mint
+{% include image-gallery.html pages = page.images_auto-swap %}
+Wallets can provide the option to automatically swap ecash from an unknown mint to their a default mint. This optional feature can simplify transactions by routing all ecash transactions through the assigned default mint. An auto-swap requires a lightning payment.
+
+{% include tip/open.html color="blue" icon="info" label="Auto-Swapping Involves Fees" %}
+
+It's important to inform users that auto-swapping involves fees and that the amount depends on the amount of ecash they want to swap and the current network conditions.
+
+{% include tip/close.html %}
+
+## Fedimint Design Best Practices
+
+### User Interface and Experience
+
+#### Federation Status Display
```

Comment:
Does Cashu not have that feature? Might be good to explain that.

Reply:
Since cashu is a single mint there is no real status it is either up and working or not, where as a fedimint could have a minatory of guardians down ans still work in a degraded state. 

Cashu does have a message of the day so it could publish a message thats its lightning node is temporarily down so minting and melting is not possible but you can still swap and funds are safe but there isnt a specific spec for this and it would just be a general message so don't think I would include that here